modules/sdk-core/src/openssl/README.md

```diff
@@ -0,0 +1,8 @@
+# Updating openssl.ts
+
+1. Clone the following [repo](https://github.com/zahin-mohammad/openssl-wasm)
+2. Use the `build.sh` script to generate the `openssl.wasm` and `openssl.js` file
+3. Convert the `openssl.js` file to a valid `openssl.ts`
```

Comment:
Not sure if it's worth it but might be nice to put a few directions on what this conversion would look like. What functions do we need to keep in the .js files? Which ones are we removing?

Reply:
Good point. I can add this in a follow up PR.

Reply:
Can probably create a patch file for this directly.